### DISCUSSION TITLE
Need help to setup project locally | getting error

### DISCUSSION THREAD

**Discussion by mdirfan40** [getredash/redash#6976] [General]
I have started server using docker server http://localhost:5001/ when i open it in browser js and css not loading 
i run yarn 
its showing error 

```
The chromium binary is not available for arm64: 
If you are on Ubuntu, you can install with: 

 apt-get install chromium-browser
```

**Comment by justinclift** [comment]
What's the system you're trying to run it on?

If it's macOS, then some of our macOS users (@gaecoli springs to mind?) can probably point you in the right direction. :smile:

**Reply by mdirfan40** [reply]
trying to run it on mac os m1

Done  Thanks
installed chromium-browser using brew
